pkgs/development/python-modules/telfhash/default.nix

```diff
@@ -23,22 +23,21 @@ buildPythonPackage rec {
   # The tlsh library's name is just "tlsh"
   postPatch = ''
     substituteInPlace requirements.txt \
-       --replace "python-tlsh" "tlsh" \
-       --replace "py-tlsh" "tlsh"
+       --replace-fail "python-tlsh" "tlsh" \
+       --replace-fail "py-tlsh" "tlsh" \
+       --replace-fail "nose>=1.3.7" ""
```

Comment:
nits: you can use `pythonRemoveDeps` here.

Reply:
I didn’t know about that, thank you! Sorry for not catching this. At least it was already present before in this case.